apps/condo/domains/marketplace/hooks/useMarketplaceInvoicesTableColumns.tsx

```diff
@@ -78,11 +88,36 @@ export const useMarketplaceInvoicesTableColumns = ({ filtersMeta }) => {
             filterDropdown: getFilterDropdownByKey(filtersMeta, 'ticket'),
             filterIcon: getFilterIcon,
         },
+        {
+            title: AddressTitle,
+            key: 'property',
+            dataIndex: 'property',
+            width: '15%',
+            render: renderAddress,
+            sorter: true,
```

Comment:
In fact, sorting by addresses will not work as expects (alphabetically). But ok, let it remain

Reply:
There is sorting in the ticket table. Maybe someone will use it